ARCs/arc-0054.md

```diff
@@ -14,35 +14,39 @@ created: 2023-09-15
 This ARC provides TEAL which would deploy a application that can be used for burning Algorand Standard Assets. The goal is to have the apps deployed on the public networks using this TEAL to provide a standardized burn address and app ID.
 
 ## Motivation
-Currently there is no official way to burn ASAs. While one can currently deploy their own app or rekey an account holding the asset to a vanity address, having a standardized address for burned assets allows explorers and dapps to display burnt supply for every ASA.
+Currently there is no official way to burn ASAs. While one can currently deploy their own app or rekey an account holding the asset to the zero address, having a standardized address for burned assets enables explorers and dapps to easily calculate and display burnt supply for any ASA burned here.
```

Comment:
> rekey an account ... to the zero address

The zero address is the only address that can not be rekeyed-to. It is interpreted as "no rekey"

Reply:
Doh!  You are right... which is why there really is no good, trustless way to use a rekeyed account as a burn location.  Fixed the commentary here.